apps/src/music/blockly/blocks/simple2.js

```diff
@@ -112,6 +116,8 @@ export const triggeredAtSimple2 = {
         name: '${block.getFieldValue(TRIGGER_FIELD)}',
         uniqueInvocationId: MusicPlayer.getUniqueInvocationId()
       };
+      var __maybeSkipSound = false;
+      var __skipSound = false;
```

Comment:
Wondering if something like `__inRandomMode` or `__skippable` would be a little clearer, just because maybeSkipSound and skipSound are a bit similar?

Reply:
Fair point.  After spending some time in this code, I'm trying out `insideRandom` and `skipSound` for the new context object, but it's easy to change again.